### DISCUSSION TITLE
Just show AR-Button

### DISCUSSION THREAD

**Discussion by sercandemirtas** [google/model-viewer#1954] [Q&A]
Hello all!

Is it possible to remove 3D view on model-viewer. I mean I just want to show AR-button. I tried with putting a picture in front of the model but sometimes it is not good for pages. I looked for it in modelviewer.dev but I couldnt find "only show ar button mode" or sth like that. 

Any suggestions or solutions would be great.

Thanks..

### Browser Affected

- [ x] Chrome
- [ x] Edge
- [x ] Firefox
- [ x] IE
- [ x] Safari

### OS

- [x ] Android
- [ x] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

**Comment by canberka** [comment]
it should be ... if it is not it can be future feature

**Comment by elalish** [comment]
Yes, it's pretty easy: https://github.com/google/model-viewer/issues/946#issuecomment-717956976

**Comment by canberka** [comment]
> Yes, it's pretty easy: [#946 (comment)](https://github.com/google/model-viewer/issues/946#issuecomment-717956976)

Easy one Inspired by model viewer means lead to problem i think .  Google 3d search button register could not sign the url for 3d button which can be seen on google search . I dont know just thinking loudly

**Comment by elalish** [comment]
@canberka I can't quite parse that; what does any of this have to do with Google search?

**Comment by canberka** [comment]
Fair enough. 

https://developers.google.com/search/docs/guides/3d-ar 
![image](https://user-images.githubusercontent.com/36857390/104771312-251e3100-5783-11eb-833b-4d389e259090.png)

**Comment by elalish** [comment]
Ah, right. Sadly this is still invitation-only. Much as I would love to see this open to all, it's out of my hands. It will only happen if we can demonstrate enough 3D models on the web, so keep doing what you're doing! If at any point this becomes more available, rest assured we'll do everything we can here to make model-viewer fit in as easily as possible.

**Reply by allenwixted** [reply]
Hi @elalish any ideas on the threshold? Have hundreds of products our eCom and retail clients are eager to get into this asap.

**Reply by elalish** [reply]
It turns out Google deals with pretty big numbers. Millions tend to look small. Still, we are trying to figure out how to make this happen. @tpsiaki, since we were just talking about this.

**Reply by aviggiano** [reply]
We're dying for an invitation too 😬 
I even tried reverse engineering their website to see if we can do it by ourselves but unfortunately it didn't work 😆
